cli/internal/cmd/iam.go

```diff
@@ -209,6 +222,7 @@ func (c *iamCreator) create(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	c.log.Debugf("Successfully created the IAM cloud resources")
 	c.cmd.Println() // Print empty line to separate after spinner ended.
```

Comment:
```suggestion
	c.cmd.Println() // Print empty line to separate after spinner ended.
	c.log.Debugf("Successfully created the IAM cloud resources")
```
Might be worth to print the debug message after we started a new line for cleaner output